src/Plugin/Condition/UserHasRole.php

```diff
@@ -38,12 +39,22 @@
 class UserHasRole extends RulesConditionBase {
 
   /**
-   * {@inheritdoc}
+   * Evaluate if user has role(s).
+   *
+   * @param \Drupal\user\UserInterface $account
+   *   The account to check.
+   * @param \Drupal\user\RoleInterface[] $roles
+   *   Array of user roles.
+   * @param string $operation
+   *   Either "AND": user has all of roles.
+   *   Or "OR": user has at least one of all roles.
+   *   Defaults to "AND".
+   *
+   * @return bool
+   *   TRUE if user has role(s).
+   *   Defaults to FALSE.
```

Comment:
Removed the doc-comment and the "return FALSE" (it's an unreachable statement, says phpstorm)